optimum/exporters/tasks.py

```diff
@@ -189,7 +189,9 @@ class TasksManager:
 
         _DIFFUSERS_TASKS_TO_MODEL_LOADERS = {
             "stable-diffusion": "StableDiffusionPipeline",
-            "stable-diffusion-xl": "StableDiffusionXLImg2ImgPipeline",
+            "stable-diffusion-xl": "StableDiffusionXLPipeline",
```

Comment:
`_optional_components` was introduced in diffusers v0.22.0 https://github.com/huggingface/diffusers/blob/v0.22.0/src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py#L144, so we would need to upgrade [`DIFFUSERS_MINIMUM_VERSION`](https://github.com/huggingface/optimum/blob/dedb852eaea3d3d899920f4017b9fd899eb2756b/optimum/utils/import_utils.py#L53) as well to avoid any potential issue when loading a SDXL model (which don't have optional components)

Reply:
[DIFFUSERS_MINIMUM_VERSION](https://github.com/huggingface/optimum/blob/dedb852eaea3d3d899920f4017b9fd899eb2756b/optimum/utils/import_utils.py#L53) was upgraded